query/inspect_test.go

```diff
@@ -18,13 +18,14 @@ import (
 	"testing"
 	"time"
 
-	"github.com/square/metrics/api/backend"
+	// "github.com/square/metrics/api/backend"
```

Comment:
I'll deal with this in the next PR. The entire thing will be around this specific file.